content/en/observability_pipelines/troubleshooting.md

```diff
@@ -65,6 +65,10 @@ If you try to install a new version of the Worker in an instance that is running
 
 If you do not see Worker logs in [Log Explorer][12], make sure they are not getting excluded in your log pipelines. Worker logs must be indexed in Log Management for optimal functionality. The logs provide deployment information, such as Worker status, version, and any errors, that is shown in the Observability Pipelines UI. The logs are also helpful for troubleshooting Worker or pipelines issues. All Worker logs have the tag `source:op_worker`.
 
+## Too many files error
+
+If you see the error `Too many files` and the Worker processes continually to restart, it could be due to a low file descriptor limit on the host. To resolve this issue for Linux environments, set `LimitNOFILE` in the systemd service configuration to `65,536` to increase the file descriptor limit.
```

Comment:
Is the phrase "the Worker processes continually to restart" right? I'm having trouble understanding it. Does it mean that the Worker processes repeatedly restart?

Reply:
Yes, I updated to say "the Worker processes repeatedly restart". Thanks!